frontend_tests/node_tests/people.js

```diff
@@ -455,17 +507,6 @@ run_test('filtered_users', () => {
 people.init();
 
 run_test('multi_user_methods', () => {
-    const emp401 = {
-        email: 'emp401@example.com',
-        user_id: 401,
-        full_name: 'whatever 401',
-    };
-    const emp402 = {
```

Comment:
For one-off users like this, I'm thinking it would be a readability tradeoff here. To fully understand the test, one would have to scroll up to find these users and see what data we're checking for.

I think that if we're checking for sort order, etc we could write a simple assert before the test such as:

```js
assert(emp1.user_id < emp2.user_id);
```

and thus explicitly declare the reasons for the test data being the way it is.

Reply:
yeah, I fixed this particular place, and I agree in general